outputs.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os/exec"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+/// The response from Slack POST
```

Comment:
The linter is asking that this comment start with "SendMessageResponse", like:
```suggestion
// SendMessageResponse is the response from Slack POST
```